README.md

```diff
@@ -50,36 +50,24 @@ pluginManagement {
 }
 ```
 
-`build.gradle`
+`build.gradle.kts`
 
-```groovy
+```kotlin
 plugins {
-    id("org.jetbrains.kotlin.jvm") version "1.9.0"
-    id("com.google.devtools.ksp") version "1.9.0-1.0.13"
+    id("org.jetbrains.kotlin.jvm")
+    id("com.google.devtools.ksp")
 }
 
 repositories {
     mavenCentral()
     google()
 }
 
-dependencies {
-    ksp("me.tatarka.inject:kotlin-inject-compiler-ksp:0.6.3")
-    implementation("me.tatarka.inject:kotlin-inject-runtime:0.6.3")
-}
-```
-
-### or with KAPT (deprecated)
-
-```groovy
-plugins {
-  id("org.jetbrains.kotlin.jvm") version "1.9.0"
-  id("org.jetbrains.kotlin.kapt") version "1.9.0"
-}
+val kotlinInjectVersion = "<latest kotlin-inject version>"
```

Comment:
Same reasoning here. It's likely that this will get out of date and cause confusion (although I do have several libraries where I update the version in the README as part of the release process).

Reply:
> although I do have several libraries where I update the version in the README as part of the release process

yep that's my standard practice, I hate it when I have to hunt down what the latest version is lol

Reply:
I've also steered away form `val whateverVersion = ` just because there's like 20 different ways to do this in gradle and I think the plain string is the easiest place to start from a copy & paste prospective.

Reply:
Here's how I keep the versions updated - https://github.com/eygraber/uri-kmp/blob/master/.github/workflows/publish_release.yml#L123

I went with the `val` just so it wasn't repeated everywhere, but if it's updated automatically it matters less.